### ISSUE TITLE
[QUESTION] ALZ vs AMBA vs AMBA-Export

### ISSUE THREAD

**Issue by RobHofmann** [anwather/amba-export#61]
Hi!

First of all, thanks for your solution.

I'm reading the following thing in the readme:
`Eventually these policies will be integrated into the ALZ project and this repo will no longer be maintained.`

Does this mean that this solution will be synced to:
https://github.com/Azure/Enterprise-Scale
or
https://github.com/Azure/azure-monitor-baseline-alerts

The first one would be a challenge (we don't use this). The last one would be alright.

Looking forward to hearing from you.

**Comment by anwather** [comment]
Yeah not sure of that direction now, the AMBA project seems to be taking on a life of its own. I need to have another look at this, but I think I was hoping that it would be integrated into Enterprise Scale and I wouldn't have to maintain a separate process.

**Comment by RobHofmann** [comment]
Hmm i understand. It sounds logical if this would become part of AMBA since its a monitoring solution, right? We have our own landingzone like setup, but that would mean we could not use this if it was in the ALZ repo.

You got any timelines on this descision? I'm a bit afraid to start using this, knowing it might be moved/changed.

**Comment by anwather** [comment]
No idea - but the way EPAC works I extract out all the policies from ALZ anyway so if this became integrated into ALZ it wouldn't be a real issue if you weren't using that repository.

**Comment by RobHofmann** [comment]
Alright, that sounds hopefull.

Looking forward to your descision. Good luck with making that :).
